### PR TITLE
Revert "Travis add mpl backend setup"

### PR DESCRIPTION
**Pull request by bsipocz**
Reverts astropy/ci-helpers#38

### CODE REVIEW

**Comment by astrofrog** [comment]
Instead of reverting, what about having an ``$XVFB`` environment variable that can be set to True? I think that would be already useful, since it's annoying to have to remember those two lines, and if they ever need to change in future, will be easier to do from one place?

**Comment by astrofrog** [comment]
(what I meant was that PySide vs Qt was too specialized, but still think that XVFB would be worthwhile)

**Comment by bsipocz** [comment]
@astrofrog - That's what I try to do, to keep the first commit from the original, but revert the second one that made it an always on feature. This is the first time I tried the revert function, so maybe it's not ideal way to do it.

**Comment by astrofrog** [comment]
@bsipocz - ok, no problem :) I would call the variable ``SETUP_XVFB`` because it can be useful not just for matplotlib.

File: README.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ Following this, various dependencies are installed depending on the following en
 * ``$DEBUG``: if `True` this turns on the shell debug mode in the install
   scripts, and provides information on the current conda install.
 
+* ``$SETUP_XVFB``: if True this makes sure e.g., interactive matplotlib
+  backends work.
```
Comment (astrofrog) [inline README.md]: add at the end of the sentence ``by starting up a X virtual framebuffer``.

**Comment by astrofrog** [comment]
:+1: apart from a small comment above :)

**Pull request merged**